### PR TITLE
add docs for UNDROP TABLE

### LINKED ISSUE #969: Add UNDROP TABLE to docs
You have 8 minutes to UNDROP a table that was dropped. Include `system.dropped_tables`

### PR DESCRIPTION
**Pull request by DanRoscigno**
Reviewers: Please check my syntax line.  I found this in a comment in the Parsers header file:
```
UNDROP TABLE [db.]name [UUID uuid]
```

But I copied the syntax line from DROP TABLE.  Let me know which one is correct and please tell me how to figure it out for myself in the future.

### Changelog category (leave one):
- Documentation (changelog entry is not required)

closes https://github.com/ClickHouse/clickhouse-docs/issues/969

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/sql-reference/statements/undrop.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+# UNDROP TABLE
+
+Cancels the dropping of the table.
+
+Beginning with ClickHouse version 23.3 it is possible to UNDROP a table 
+within 8 minutes of issuing the DROP TABLE statement.  Dropped tables are listed in 
+a system table called `system.dropped_tables`.
+
```
Comment (tavplubix) [inline docs/en/sql-reference/statements/undrop.md:13]: It worth mentioning that the query works with `Atomic` databases only

Comment (tavplubix) [reply docs/en/sql-reference/statements/undrop.md:13]: Also I've just realized that it does not undrop the inner table of an MV (for MVs without `TO` clause), it has to be undropped manually

File: docs/en/sql-reference/statements/undrop.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+:::
+
+:::tip
+Also see [DROP TABLE](/docs/en/sql-reference/statements/drop.md)
+:::
+
+Syntax:
+
+``` sql
+UNDROP [TEMPORARY] TABLE [IF EXISTS] [db.]name [ON CLUSTER cluster] [SYNC]
```
[the remark this answers is not in the document]
Comment (DanRoscigno) [reply docs/en/sql-reference/statements/undrop.md:28]: thanks for that clarification!

File: docs/en/sql-reference/statements/undrop.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+slug: /en/sql-reference/statements/undrop
+sidebar_label: UNDROP
+---
+
+# UNDROP TABLE
+
+Cancels the dropping of the table.
+
+Beginning with ClickHouse version 23.3 it is possible to UNDROP a table
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/undrop.md:10]:
```suggestion
Beginning with ClickHouse version 23.3 it is possible to UNDROP a table in an Atomic database
```

File: docs/en/sql-reference/statements/undrop.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+# UNDROP TABLE
+
+Cancels the dropping of the table.
+
+Beginning with ClickHouse version 23.3 it is possible to UNDROP a table in an Atomic database
+within `database_atomic_delay_before_drop_table_sec` (8 minutes by default) of issuing the DROP TABLE statement.  Dropped tables are listed in 
+a system table called `system.dropped_tables`.
+
+If you have a materialized view without a `TO` clause associated with the dropped table, then you will also have to UNDROP the inner table of that view.
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/undrop.md:14]: @tavplubix please can you check this information about also undropping the MV table?

**Review by tavplubix — APPROVED** [review]

**Comment by DanRoscigno** [comment]
Thanks so much for your help @tavplubix !

**Pull request merged**